test/Integration.t.sol

```diff
@@ -150,6 +180,10 @@ contract GeneralisedTests is CapAutomatorIntegrationTestsBase {
             uint256 postIncreaseSupplyCap = pool.getReserveData(assets[i]).configuration.getSupplyCap();
 
             assertEq(postIncreaseSupplyCap, currentSupply + newGap);
+
+            ( ,,, uint48 lastUpdateBlock, uint48 lastIncreaseTime ) = capAutomator.supplyCapConfigs(assets[i]);
+            assertEq(lastUpdateBlock,  block.number);
+            assertEq(lastIncreaseTime, block.timestamp);
```

Comment:
Same here. These should be asserted to be different values before `exec` is called.